documentation/glossary.md

```diff
@@ -0,0 +1,56 @@
+# Glossary of aviation terms
+*Openscope uses some terms specific to aviation, this glossary attempts to define them.*
+---
+## ILS
+*abbr. of* Instrument Landing System
+
+A radio antenna at the end of the runway, known as a ''localizer'', guides pilots down onto the runway for landing, via an instrument known as a *guide scope*.
+
+## Fix
+
+A pre-defined point that describes an aircraft's current position. A fix can be an airport (`EGKK`), a waypoint (`WILLO`), or other points, such as a VOR station (`GWC`).
+
+## Waypoint
+
+A fix that is used to aid in navigation, defined in a flight plan or route. Waypoints are assigned a 5-letter code, such as `WILLO` and `ABTUM`.
+
+## Flight Level
+*abbr.* FL
+
+A plane's altitude, in 100s of feet. For example, a plane at `FL350` is flying at 35,000ft.
+
+## SID
+*abbr. of* Standard Instrument Departure
+
+A route that an aircraft will follow to exit the airport's radar control area. Each SID begins at the airport and terminates at a fix, with waypoints in between.
+
+## STAR
+*abbr. of* Standard Terminal Arrival Route
+
+A route that an aircraft will follow to approach the airport, the inverse of a SID.
+
+## Airspace
+
+A pre-defined three-dimensional area where certain restrictions occur. For example, in Class A airspace, VFR aircraft are not permitted.
+
+## VFR
+*abbr. of* Visual Flight Rules
+
+VFR aircraft do not have instruments enabling them to navigate and land under ILS. As such, they rely on visual clues and ATC instruction to navigate and land.
```

Comment:
I wouldn't exactly involve aircraft in this, but up to @erikquinn since they are a set of _rules_, instead I think this should be based of the rules, since VFR aircraft can have instruments allowing them to navigate

Reply:
done